akka-stream/src/main/scala/akka/stream/scaladsl/Hub.scala

```diff
@@ -741,15 +744,15 @@ object PartitionHub {
    *   identifiers and the second is the stream element. The function should return the selected consumer
    *   identifier for the given element. The function will never be called when there are no active consumers,
    *   i.e. there is always at least one element in the array of identifiers.
-   * @param startAfterNbrOfConsumers Elements are buffered until this number of consumers have been connected.
+   * @param startAfterNrOfConsumers Elements are buffered until this number of consumers have been connected.
    *   This is only used initially when the stage is starting up, i.e. it is not honored when consumers have
    *   been removed (canceled).
    * @param bufferSize Total number of elements that can be buffered. If this buffer is full, the producer
    *   is backpressured.
    */
-  def statefulSink[T](partitioner: () ⇒ (ConsumerInfo, T) ⇒ Long, startAfterNbrOfConsumers: Int,
+  def statefulSink[T](partitioner: () ⇒ (ConsumerInfo, T) ⇒ Long, startAfterNrOfConsumers: Int,
```

Comment:
@ktoso what do you think about this? I don't mind introducing it as `ApiMayChange`